client/js/components/statement/fragment/Version.vue

```diff
@@ -86,13 +86,13 @@
 
 <script>
 import { checkResponse, CleanHtml, dpApi, DpLoading, formatDate } from '@demos-europe/demosplan-ui'
-import FragmentStatus from './Status'
+import DpFragmentStatus from './Status'
 
 export default {
   name: 'DpFragmentVersion',
 
   components: {
-    'dp-fragment-status': FragmentStatus,
+    'dp-fragment-status': DpFragmentStatus,
```

Comment:
can now be simplified to:

```suggestion
    DpFragmentStatus,
```